pkg/controller/servingcert/controller/secret_creating_controller_test.go

```diff
@@ -147,6 +147,7 @@ func TestServiceServingCertControllerSync(t *testing.T) {
 			expectedSecretAnnotations: map[string]string{
 				api.AlphaServiceUIDAnnotation:  testServiceUID,
 				api.AlphaServiceNameAnnotation: testServiceName,
+				api.OwningJiraComponent:        api.OwningJiraComponent,
```

Comment:
Wrong annotation name here as well; `apiannotations.OpenShiftComponent`. Several occurences in this file.